lib/__tests__/notify.ts

```diff
@@ -50,7 +50,9 @@ describe('lib/notify', () => {
             resolve();
           })
         );
-        output = `${output}${new Date(now).toISOString()} ${actionType}: ${filePath}\n`;
+        output = `${output}${new Date(
```

Comment:
Uh oh, is prettier acting up again?

Reply:
I'm hoping this is just a one-off thing and not the sign of this acting up 🤞